tests/core/trt_ONNX/logger.py

```diff
@@ -0,0 +1,120 @@
+# ! /usr/bin/python
```

Comment:
Why are you defining a new logger?

Reply:
yeah, I saw that and actually had the same question...

Reply:
Here we go:
https://github.com/NVIDIA/NeMo/pull/409/files#diff-b9abacf00e0d4de047de1bb43651cfedR366
```
         with trt.Builder(TRT_LOGGER) as builder, network, parser:
```
This is the way TRT implemented. They need logger to be inherited from theirs:
https://github.com/NVIDIA/TensorRT/blob/master/include/NvInfer.h#L6784

Reply:
I see that you are using TRT_LOGGER but that seems to be completely separate from G_LOGGER. I am completely fine with the use of TFT_LOGGER.

My question is related to why G_LOGGER exists. Can you explain why you cannot use nemo.logging instead of G_LOGGER?

Reply:
OK, I'll refactor it to use nemo.logging